src/main/java/org/folio/sidecar/service/routing/IngressRequestHandler.java

```diff
@@ -46,8 +45,7 @@ private void forwardRequest(RoutingContext rc) {
     headers.set(OkapiHeaders.URL, sidecarProperties.getUrl());
     headers.set(OkapiHeaders.MODULE_ID, moduleProperties.getId());
 
-    var addModuleNameToPath = TRUE.equals(rc.get(RoutingUtils.ADD_MODULE_NAME_TO_PATH_KEY));
-    var path = RoutingUtils.buildPathWithPrefix(rc, addModuleNameToPath, moduleProperties.getName());
+    var path = buildPathWithPrefix(rc, sidecarProperties.isModulePrefixEnabled(), moduleProperties.getName());
```

Comment:
Is it a single place for all requests?
Do we have something related to the ModulePrefixEnabled in ForwardRequestService?

Reply:
Yes, for egress request information is taken from module bootstrap information (module discovery in mgr-applications), there is no need to update path